servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -180,21 +181,13 @@ private GetNamedRefsParams getGetNamedRefsParams(boolean fetchMetadata) {
             if (commit == null) {
               commit = CELUtil.EMPTY_COMMIT_META;
             }
-            String refType;
-            if (reference instanceof Branch) {
-              refType = "BRANCH";
-            } else if (reference instanceof Tag) {
-              refType = "TAG";
-            } else {
-              refType = "REFERENCE";
-            }
             return script.execute(
                 Boolean.class,
                 ImmutableMap.of(
                     VAR_REF,
                     reference,
                     VAR_REF_TYPE,
-                    refType,
+                    reference.getType().name().toUpperCase(Locale.ROOT),
```

Comment:
This should not be necessary, I guess, if enum names were upper case.

Reply:
I agree, I think it would be nicer to rather do `reference.getType().name().toLowerCase()` in the places where the type is being used in the URI